examples/uid_higher_than_zero.yaml

```diff
@@ -0,0 +1,8 @@
+name: uid_higher_or_equal_than_zero
+description: This policy traces read and write syscall for uids higher or equal than 0
+scope: 
+  - uid>=1000
+defaultAction: audit
+rules:
+  - event: read
+  - event: write
```

Comment:
```suggestion
name: uid_higher_than_zero
description: This policy traces read and write syscalls for uids higher than 0
scope: 
  - uid>0
defaultAction: audit
rules:
  - event: read
  - event: write

```

Reply:
Here we want to change only the file name, the policy should still use `>=`